src/scss/base/_base.scss

```
@@ -1,16 +1,200 @@
-// Base styles
+* {
+  outline: transparent;
+}
 
 *,
-*::before,
-*::after {
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+html {
   margin: 0;
   padding: 0;
+  font-family: 游ゴシック体, 'Yu Gothic', YuGothic, 'ヒラギノ角ゴシック Pro', 'Hiragino Kaku Gothic Pro', メイリオ, Meiryo, Osaka, 'ＭＳ Ｐゴシック', 'MS PGothic', sans-serif;
+  font-weight: normal;
+  color: #000000;
+  font-size: 14px;
+  line-height: 1.6;
+  background: #eaf1f4;
+}
+
+* {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
   box-sizing: border-box;
 }
 
-html,
 body {
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  scroll-behavior: smooth;
+  margin: 0px;
+  position: relative;
+}
+
+h1,
+h2,
+h3,
+h4 {
+  font-weight: normal;
+  margin: 0px;
+}
+
+h2 {
+  line-height: 1.3;
```

Please define variable for this case in this file. 
move all variable local on the top of this file
`$h2-line-height: 1.3`

Please do other case below same this case!

```suggestion
  line-height: $h2-line-height;
```